.github/workflows/create-release-branch.yml

```diff
@@ -1,19 +1,18 @@
-# Bump package version numbers and create a release branch
-name: Release branch - Create
+# create release branch from pre-release branch
+name: Create release branch from pre-release branch
 
 on:
-  # Allows you to run this workflow manually from the Actions tab
   workflow_dispatch:
     inputs:
       branch:
-        description: 'Branch or tag to create release from'
+        description: 'Pre-release branch to release from'
         required: true
-        default: 'main'
 
 jobs:
-  release:
-    name: Bump versions and create changelog PR
-    runs-on: ubuntu-latest
+  create_release:
```

Comment:
Should we check on top if the base branch is of the right format `prerelease-**` otherwise abort (with failure).